website/src/components/DataUseTerms/EditDataUseTermsModal.tsx

```diff
@@ -47,8 +47,7 @@ type LoadedState = {
     earliestRestrictedUntil: DateTime | null;
 };
 
-// eslint-disable-next-line @typescript-eslint/no-explicit-any -- TODO(#3451) use proper types
-function getLoadedState(rows: Record<string, any>[]): LoadedState {
+function getLoadedState(rows: Record<string, string | number | boolean | null>[]): LoadedState {
```

Comment:
This data is coming from the LAPIS details endpoint, isn't it? We already have a type for that:
```suggestion
function getLoadedState(rows: Details[]): LoadedState {
```
fromk `import type {Details} from "../../types/lapis.ts";`

Reply:
https://github.com/loculus-project/loculus/pull/3464/commits/b817828476b2acfab01801f968fe73cae9dca8ad